.github/workflows/windows-conda.yml

```diff
@@ -47,9 +47,14 @@ jobs:
           ..
         if errorlevel 1 exit 1
 
+        :: Change path to find eigenpy DLL when creating stub
+        set OLD_PATH=%PATH%
+        set PATH=%PATH%;%CD%
         :: Build and Install
         cmake --build . --config Release --target install
         if errorlevel 1 exit 1
+        :: Set old path
+        set PATH=%OLD_PATH%
 
         :: Build stubs
         git clone https://github.com/jcarpent/pybind11-stubgen.git
```

Comment:
I think the manual stub generation is not mandatory anymore.

Maybe, a better fix should be to modify the path in jrlcmakemodule when calling the stub generator.